api/DataTransfer.json

```diff
@@ -789,10 +794,10 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": true
+              "version_added": "3.5"
```

Comment:
And `mozUserCancelled `: https://github.com/mozilla/gecko-dev/blob/e8d5a5ad644903b001c076279703a7a8f8a811ea/dom/webidl/DataTransfer.webidl#L153